### PR TITLE
Add business-status and authored-on fields to EPSAT search page.

### PR DESCRIPTION
**Pull request by mfjarvis**

### CODE REVIEW

**Review by callumtipper-IW — DISMISSED** [review]

File: tool/site/client-react/src/components/selectField.tsx [hunk 1]
```diff
@@ -1,11 +1,10 @@
-import React, {FC} from "react"
+import React, {FC, HTMLProps} from "react"
 import {Select} from "nhsuk-react-components"
 import {Field} from "formik"
 import {Coding} from "fhir/r4"
+import {FormElementProps} from "nhsuk-react-components/dist/util/types/FormTypes"
```
Comment (dan-jones-iw) [inline tool/site/client-react/src/components/selectField.tsx:5]: can we import this from same place as line 2?

File: tool/site/server_hapi/src/services/communication/live-eps-client.ts [hunk 2]
```diff
@@ -11,9 +11,17 @@ export class LiveEpsClient implements EpsClient {
...
   }
 
-  async makeGetTrackerRequest(query: Record<string, string>): Promise<Bundle | OperationOutcome> {
-    const queryStr = new URLSearchParams(query).toString()
-    return await (await this.makeApiCall<Bundle | OperationOutcome>(`Task?${queryStr}`)).data
+  async makeGetTrackerRequest(query: Record<string, string | Array<string>>): Promise<Bundle | OperationOutcome> {
+    const urlSearchParams = new URLSearchParams()
+    Object.keys(query).forEach(key => {
+      const valueOrValues = query[key]
+      if (typeof valueOrValues === "string") {
```
Comment (dan-jones-iw) [inline tool/site/server_hapi/src/services/communication/live-eps-client.ts:18]: do we have access to `toArray` here?

**Review by dan-jones-iw — APPROVED** [review]

File: tool/site/client-react/src/components/selectField.tsx (hunk 1 above)
Comment (mfjarvis) [reply tool/site/client-react/src/components/selectField.tsx:5]: No

File: tool/site/server_hapi/src/services/communication/live-eps-client.ts (hunk 2 above)
Comment (mfjarvis) [reply tool/site/server_hapi/src/services/communication/live-eps-client.ts:18]: No

**Review by callumtipper-IW — APPROVED** [review]

**Pull request merged**